Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -25,6 +78,42 @@ def parse_resource_ids(resource_id: str | None) -> list[str]:
     return resource_ids
 
 
+def parse_filter_field(filter_string: str | None):
+    """
+    Parses a list representation of name and values with the form of 'name=<name>,values=<values>.
+    You can specify up to 50 filters and up to 200 values per filter in a single request.
+    Filter strings can be up to 255 characters in length.
+    Args:
+        filter_list: The name and values list
+    Returns:
+        A list of dicts with the form {"Name": <key>, "Values": [<value>]}
+    """
+    filters = []
+    list_filters = argToList(filter_string, separator=";")
+    if len(list_filters) > MAX_FILTERS:
+        list_filters = list_filters[0:50]
+        demisto.debug("Number of filter is larger then 50, parsing only first 50 filters.")
+    regex = re.compile(
```

Comment:
In general it is better to avoid regex, if we can split by a comma and use split again to get the values it is better.

Reply:
I know but according to the tag [docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html) filter [docs](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSecurityGroups.html) the tag can be with a comma, so it is problematic to use split by comma.